conceptual/EFCore.PG/mapping/nts.md

```diff
@@ -88,37 +88,52 @@ Note that the plugin is far from covering all spatial operations. If an operatio
 .NET                             | SQL
 ---------------------------------|-----
 geom.Area()                      | [ST_Area(geom)](https://postgis.net/docs/manual-3.0/ST_Area.html)
+geom.AsBinary()                  | [ST_AsBinary(geom)](https://postgis.net/docs/manual-3.0/ST_AsBinary.html)
 geom.AsText()                    | [ST_AsText(geom)](https://postgis.net/docs/manual-3.0/ST_AsText.html)
 geom.Boundary                    | [ST_Boundary(geom)](https://postgis.net/docs/manual-3.0/ST_Boundary.html)
+geom1.Centroid(geom2))           | [ST_Centroid(geom1, geom2)](https://postgis.net/docs/manual-3.0/ST_Centroid.html)
 geom1.Contains(geom2))           | [ST_Contains(geom1, geom2)](https://postgis.net/docs/manual-3.0/ST_Contains.html)
+geomCollection.Count             | [ST_NumGeometries(geom1)](https://postgis.net/docs/manual-3.0/ST_NumGeometries.html)
```

Comment:
We also translate Count on line strings (to ST_NumPoints)

Reply:
Also added in the latest commit, as well as the operations found in [this file](https://github.com/npgsql/efcore.pg/blob/main/src/EFCore.PG.NTS/Query/ExpressionTranslators/Internal/NpgsqlNetTopologySuiteMethodCallTranslatorPlugin.cs).